### ISSUE TITLE
Relationship chart - demo-dev

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#4729]
It should be possible to define the closest relationships as the default relationship.

All the possible relations takes a long time on my site.

**Comment by FrankWarius** [comment]
No, this is not the solution.

Setting Recursion to none and selecting "Find other relationships" and shows in 4 paths this example https://wbt.warius.info/tree/Warius/relationships-0-1/I1211/I12 
Selection "Find the closest relationships" shows as indented one path https://wbt.warius.info/tree/Warius/relationships-0-0/I1211/I12
but only setting Recursion to "3" in control panel - what a normal user can't do (only by manipulating the URL) shows more relationships https://wbt.warius.info/tree/Warius/relationships-0-3/I1211/I12 (takes a long time)

My normal use case is first selecting "Find the closest relationships" to proof if there is at least one relationship and then, if needed, start deeper investigation.

So if it "Find the closest relationships" is default (by code or selectable in control panel) it would be easier to me.

**Comment by melizaa** [comment]
I filled now Recursion None.

As Frank writes, this is not the solution.

"Find any relationship" is selected and I see one result.
But, there is no option to return all the paths.

The additional selection is only "Find relationships via ancestors".

When I selected Recursion Unlimited, "Find relationships via ancestors" was selected, but I had also an option to select "Find the closest relationships".

I want to see both options, but with "Find the closest relationships" as the default.
